quantestpy/converter/all.py

```diff
@@ -0,0 +1,28 @@
+from quantestpy import QuantestPyCircuit
+from quantestpy.converter.qasm_and_qiskit import (
+    _cvt_openqasm_to_quantestpy_circuit, _cvt_qiskit_to_quantestpy_circuit,
+    _is_instance_of_qiskit_quantumcircuit)
+from quantestpy.exceptions import QuantestPyError
+
+
+def cvt_all_circuit_to_quantestpy_circuit(circuit) -> QuantestPyCircuit:
```

Comment:
I want to change the function name into `cvt_input_circuit_to_quantestpy_circuit` or simply `cvt_to_quantestpy_circuit`. Accordingly, `all.py` will be changed into `converter_to_quantestpy_circuit.py` or simply `converter.py`.

It is confusing to set `all.py` and `qasm_and_qiskit.py` in same directory.
It may be better to set `qasm_and_qiskit.py` in `converter/language` (or another directory name).
Moreover, in my sense, I want to split `qasm_and_qiskit.py` into `qasm.py` and `qiskit.py` because cirq users want to use `_cvt_openqasm_to_test_circuit`.

Reply:
- I renamed `cvt_all_circuit_to_quantestpy_circuit(circuit)` -> `cvt_input_circuit_to_quantestpy_circuit(circuit)`
- I renamed `all.py` -> `converter_to_quantestpy_circuit.py`
- I added a directory `sdk`
- I split `qasm_and_qiskit.py` into `qasm.py` and `qiskit.py`